policy-engine/src/graph.rs

```diff
@@ -39,6 +42,8 @@ pub(crate) async fn index(
     req: HttpRequest,
     app_data: actix_web::web::Data<AppState>,
 ) -> Result<HttpResponse, GraphError> {
+    let _span = get_tracer().start("index", None);
```

Comment:
also `_` maybe here? See above

Reply:
We need a name here so that async call on line 58 could be instrumented

Reply:
Oh, in that case we can remove the `_` prefix since it's actually used.

```suggestion
    let span = get_tracer().start("index", None);
```